include/boost/url/grammar/parse.hpp

```diff
@@ -77,216 +63,166 @@ parse(
     error_code& ec,
     char ch) noexcept;
 
-/** @brief Parse a sequence of elements
+/** Parse two or more sequential grammar rules
```

Comment:
How about just "Parse a range of characters"

Reply:
OK. I like all of them:

- "substring" or "range of characters" end up being synonymous here. 
- "string" could imply "substring" in this context
-  even "" seems ok as parsing a grammar rule already implies there's a string being parsed